backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchIndexer.java

```diff
@@ -99,7 +99,9 @@ public ElasticSearchIndexer(Client client, DeleteByQueryPerformer deleteByQueryP
     
     public IndexResponse indexMessage(String id, String content) {
         checkArgument(content);
-        LOGGER.debug(String.format("Indexing %s: %s", id, content));
```

Comment:
You can do better:
```
        LOGGER.debug("Indexing {}: {}", id, content);
```

believing to the doc it won't compute instead in debug mode:
```
  /**
   * Log a message at the DEBUG level according to the specified format
   * and arguments.
   * <p/>
   * <p>This form avoids superfluous object creation when the logger
   * is disabled for the DEBUG level. </p>
   *
   * @param format the format string
   * @param arg1   the first argument
   * @param arg2   the second argument
   */
  public void debug(String format, Object arg1, Object arg2);
```